docs-new/docs/contribute/docs/source-file-structure.md

```diff
@@ -1,149 +1,108 @@
-  from Keptn v1
+the `docs-new/docs` directory in the repository.
+
+The subdirectories with content are:
+
+- `assets`: This folder is used to save assets such as code examples that are used throughout the documentation.
+  Many sub folders also contain an `assets` folder to keep asset files closer to the content where they are referenced.
+- `components`: Information about the different subcomponents of Keptn
+- `contribute`: Contains information on how to contribute to the different parts of Keptn
+- `core-concepts`: A brief overview of Keptn, its features and use cases, and its history
+- `getting-started`: Hands-on exercises that demonstrate the capabilities of Keptn
+- `guides`: Guides and how-to material about using Keptn features
+- `installation`: Requirements and instructions for installing and enabling Keptn
+- `migrate`: Information to help users who are migrating to Keptn from Keptn v1
+- `reference`: Reference pages for the CRDs and APIs that Keptn provides
+- `use-cases`: Examples and exercises of using Keptn in different scenarios
 
 ### Working with reference pages
 
 The Keptn documentation includes two reference sections
 that document the Keptn APIs and CRDs.
 For background information, see:
 
-* [Kubernetes API Concepts](https://kubernetes.io/docs/reference/using-api/api-concepts/)
-* [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/)
+- [Kubernetes API Concepts](https://kubernetes.io/docs/reference/using-api/api-concepts/)
+- [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/)
 
 #### API Reference
 
 The
 [API Reference](../../reference/api-reference/index.md)
-pages are autogenerated from the source code.
-This is a comprehensive list of all APIs and resources Keptn uses,
-most of which users seldom see.
+pages are autogenerated from source code.
+This is a comprehensive list of all APIs and resources Keptn uses.
 
 Descriptive text for the APIs is authored in the source code itself.
-Each operator has its own API library with different versions.
+Each operator has its own API with different versions.
 The source locations are:
 
-* [Lifecycle API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/lifecycle)
-* [Metrics API](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/api)
-* [Options API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/options)
+- [Lifecycle API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/lifecycle)
+- [Metrics API](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/api)
+- [Options API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/options)
 
 The text is coded in a limited form of markdown.
 
 To regenerate the autogenerated API reference docs,
 execute the following script
-in the root directory of your `lifecycle-toolkit` clone:
+in the root directory of the repository:
```

Comment:
This sounds like this must be done in the repository itself rather than in ones local clone.  I think it may confuse new users.